spec/airbrake_spec.rb

```diff
@@ -6,7 +6,7 @@
   end
 
   before do
-    described_class.configure do |c|
+    @notifier = described_class.configure do |c|
```

Comment:
Cached only within that test https://www.relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let